### PR TITLE
Add OKX icon

### LINKED ISSUE #10371: Request: OKX
### Brand Name

OKCoin

### Website

https://www.okcoin.com/

### Popularity Metric

Global rank of 78,000

### Official Resources for Icon and Color

None

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by dvislov**
image

**Issue:** closes #10371

**Popularity metric:** Global rank 1500 https://www.similarweb.com/website/okx.com/#overview

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description
The official logo (https://en.wikipedia.org/wiki/OKX) contains 3 letters, but the most-known icon is only the last X letter consisting of 5 squares. It is used on smartphone application icons, favicons, etc.
https://www.okx.com/learn/okex-rebrands-to-okx  
The color `#121212` is taken from its site background styles
image
image

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -10742,6 +10742,11 @@
             "source": "https://www.okta.com/press-room/media-assets/",
             "guidelines": "https://www.okta.com/terms-of-use-for-okta-content/"
         },
+        {
+            "title": "OKX",
+            "hex": "121212",
```
Comment (uncenter) [inline _data/simple-icons.json:10747]:
```suggestion
            "hex": "000000",
```

I'm seeing `#000000` as the primary background of the website and as the background color of the favicon and other promotional materials.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -10742,6 +10742,11 @@
...
+            "source": "https://www.okx.com"
```
Comment (uncenter) [inline _data/simple-icons.json:10748]: I'm not seeing any SVGs on this website - did you get it from https://commons.wikimedia.org/wiki/File:OKX_logo.svg?

Comment (service-paradis) [reply _data/simple-icons.json:10748]: Favicon and [application icon](https://www.okx.com/fr/download) are also using rounded corners:
image

File: _data/simple-icons.json (hunk 1 above)
Comment (dvislov) [reply _data/simple-icons.json]: Fixed

File: _data/simple-icons.json (hunk 2 above)
Comment (dvislov) [reply _data/simple-icons.json:10748]: @uncenter Yes, I took the logo from the wiki and cut it in Inkscape
@service-paradis you are right, round corners added in ratio from application icon

**Review by uncenter — CHANGES_REQUESTED** [review]
I don't think it makes sense to go with the application icon here, and the Wikimedia Commons source doesn't even match up as it doesn't have the rounded corners. Let's just go with the full "OKX" text please?

**Comment by uncenter** [comment]
> I don't think it makes sense to go with the application icon here, and the Wikimedia Commons source doesn't even match up as it doesn't have the rounded corners. Let's just go with the full "OKX" text please?

Still not seeing the OKX text I mentioned earlier.

**Comment by uncenter** [comment]
@dvislov are you still interested in working on this pull request?

**Comment by uncenter** [comment]
Closing as no response from PR author, can reopen if you have still have an interest in working on it! Future contributors- feel free to pick up where this left off in a new PR.

**Pull request closed without merging**